### PR TITLE
Update LICENSE

### PR DESCRIPTION
**Pull request by lexvanderstoep**
Update the year to reflect the present.

### CODE REVIEW

**Comment by lexvanderstoep** [comment]
Bit of a nit, but noticed the license was off.

File: LICENCE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2007-2023 Nils Anders Danielsson, Ulf Norell, Shin-Cheng
+iCopyright (c) 2007-2024 Nils Anders Danielsson, Ulf Norell, Shin-Cheng
```
Comment (gallais) [inline LICENCE:1]: What's with the `i`?

```suggestion
Copyright (c) 2007-2024 Nils Anders Danielsson, Ulf Norell, Shin-Cheng
```

**Review by jamesmckinna — CHANGES_REQUESTED** [review]
Thanks for the nitpicking! 
Approve subject to following @gallais 's suggestion!

File: LICENCE (hunk 1 above)
Comment (lexvanderstoep) [reply LICENCE]: I think that's a remnant of me using Emacs for the first time after using Vim for a while... Fixed.

**Review by jamesmckinna — APPROVED** [review]

**Pull request merged**